desktop/cmp/button/Button.scss

```diff
@@ -370,3 +370,22 @@
     }
   }
 }
+
+//------------------------
+// Header "tool" buttons in panels and docks
+// Adjustment needed only in light theme.
+//------------------------
+*:not(.xh-dark),
+*:not(.bp3-dark) {
```

Comment:
Do you know if we need the `:not(.bp3-dark)` here?  Is there a case where that would be applied but `.xh-dark` is not?

Reply:
@amcclain I've removed `:not(.bp3-dark)`.  I had just thrown it in because it seemed to be a pattern up higher.